config/prometheus/alerts.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: alerting-rules
+  namespace: system
+spec: 
+  groups:
+    - name: alerts
+      rules:
+      - record: ramen_sync_duration_seconds
+        expr: (time() - (ramen_last_sync_timestamp_seconds{job='ramen-hub-operator-metrics-service'} > 0))
+      - record: ramen_rpo_difference
+        expr: ramen_sync_duration_seconds / on(policyname) group_left() (ramen_policy_schedule_interval_seconds{job="ramen-hub-operator-metrics-service"})
```

Comment:
These are recording rules and not alerting rules. So, it's better to group these separately as "recording rules".